resources/js/Pages/Admin/Dashboard.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <v-container>
-    Hello this is where you'll see statistics about your project.
+    <v-card evaluation="5">
+      <v-card-text> {{ $t('comps.msg2') }}</v-card-text>
```

Comment:
please use a new translation file `msgs.js` for such messages. `comps` is shot for components which should only have plural and singular names of a component for example:
```
post: {
  p: 'Posts',
  s: 'post',
}
```